cpp/src/barretenberg/bb/get_crs.hpp

```diff
@@ -0,0 +1,84 @@
+#pragma once
+#include "exec_pipe.hpp"
+#include "file_io.hpp"
+#include "vinfo.hpp"
+#include <filesystem>
+#include <fstream>
+#include <ios>
+#include <barretenberg/ecc/curves/bn254/g1.hpp>
+#include <barretenberg/srs/io.hpp>
+
+inline std::vector<uint8_t> download_g1_data(size_t num_points)
+{
+    size_t g1_start = 28;
+    size_t g1_end = g1_start + num_points * 64 - 1;
+    std::string command = "curl -s -H \"Range: bytes=" + std::to_string(g1_start) + "-" + std::to_string(g1_end) +
+                          "\" 'https://aztec-ignition.s3.amazonaws.com/MAIN%20IGNITION/monomial/transcript00.dat'";
```

Comment:
Would we need to consider transcript01.dat..transcript02.dat as well ?

Reply:
Good call, this won't work beyond 2^19 for now.